web/docs/tql2/operators/compress_brotli.md

```diff
@@ -0,0 +1,48 @@
+# compress_brotli
+
+Compresses a stream of bytes using Brotli compression.
+
+```tql
+compress_brotli [level=int, window_bits=int]
+```
+
+## Description
+
+The `compress_brotli` operator compresses bytes in a pipeline incrementally.
+
+:::note Streaming Compression
+The operator uses [Apache Arrow's compression
+utilities][apache-arrow-compression] under the hood, and transparently supports
+all options that Apache Arrow supports for streaming compression.
+:::
+
+[apache-arrow-compression]: https://arrow.apache.org/docs/cpp/api/utilities.html#compression
```

Comment:
```suggestion
```

I think the user doesn't care if we expose all available options.

(This applies to all docs pages.)